resources/dicts/events/death/general.json

```diff
@@ -5426,7 +5328,7 @@
         "sub_type": [ "murder" ],
         "tags": [],
         "weight": 20,
-        "event_text": "m_c dies during battle training with r_c, r_c exclaims that it was all an accident.",
+        "event_text": "m_c died during battle training with r_c, r_c exclaimed that it was all an accident.",
```

Comment:
I'd add a period after the first r_c and replace the second r_c with a pronoun tag.